include/swift/AST/DiagnosticsSema.def

```diff
@@ -167,9 +167,9 @@ ERROR(init_candidate_inaccessible,none,
 
 ERROR(candidate_from_missing_import,none,
       "%0 %1 is not available due to missing import of defining module %2",
```

Comment:
General comment as this wasn't introduced in this PR. Should we try to avoid `available` here as it points to the availability system, maybe `reachable` or `visible`?

Reply:
Yeah, that's good feedback. Perhaps we should use the word "inaccessible" which would match the wording of diagnostics about declarations that are unavailable due to access levels.

Reply:
Yeah, this should not block this PR. And "inaccessible" sounds good to me.